src/hooks/useContractConfig.ts

```
@@ -1,13 +1,15 @@
 import { useAccount } from 'wagmi';
 import { NETWORK_CONFIG } from '../constants/networks';
+import useBalance from './useBalance';
 
 const useContractConfig = () => {
 	const account = useAccount();
 	const chainId = account?.chainId;
 	const network = chainId ? NETWORK_CONFIG[chainId] : undefined;
 	const isWrongNetwork = !network;
+	const balance = network && account.address ? useBalance(network, account.address) : undefined;
```

ESLint

[error] React Hook "useBalance" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)